### ISSUE TITLE
calicoctl fails when TPR does not exist

### ISSUE THREAD

**Issue by heschlie** [projectcalico/libcalico-go#477]
When calicoctl is run against KDD and a TPR (GlobalBGPConfig in this case) is missing it fails to run.

## Expected Behavior

If the TPR is missing it should be created, or at least `EnsureInitialized()` should be called first.

## Current Behavior

```
$ ./calicoctl config set nodeToNodeMesh off
Error executing command: resource does not exist: GlobalBGPConfig(name=NodeMeshEnabled)
```

## Possible Solution

Perhaps call `EnsureInitialized()` before reaching out to resources?

## Steps to Reproduce (for bugs)

1. Stand up k8s API server or cluster
2. Call `./calicoctl config set nodeToNodeMesh off` with master build of calicoctl

**Comment by robbrockbank** [comment]
Probably not super urgent as a calico deployment is likely to be running calico/node prior to configuring the TPRs (and the calico/node will create the TPRs as part of  initialization).

**Comment by bcreane** [comment]
Now that we pre-create the policy/CRDs via manifest, we will not encounter this issue. Once Gunjan's fix for migrating TPR->CRD is complete, we can mark this resolved.

**Comment by gunjan5** [comment]
fixed in Calico v2.5.0 (libcalico-go v1.6.0)
